packages/cryptoplease/lib/features/payment_request/src/presentation/link_details/share_request.dart

```diff
@@ -0,0 +1,90 @@
+import 'package:auto_route/auto_route.dart';
+import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
+import 'package:qr_flutter/qr_flutter.dart';
+import 'package:share/share.dart';
+
+import '../../../../../core/presentation/format_amount.dart';
+import '../../../../../core/presentation/utils.dart';
+import '../../../../../core/tokens/token_list.dart';
+import '../../../../../di.dart';
+import '../../../../../l10n/l10n.dart';
+import '../../../../../ui/app_bar.dart';
+import '../../../../../ui/button.dart';
+import '../../../../../ui/colors.dart';
+import '../../../../../ui/content_padding.dart';
+import '../../../../../ui/rounded_rectangle.dart';
+import '../../../../../ui/share_message/share_message_bubble.dart';
+import '../../../../../ui/share_message/share_message_header.dart';
+import '../../../../../ui/tab_bar.dart';
+import '../../../../../ui/theme.dart';
+import '../../bl/payment_request.dart';
+
+part 'components/share_link.dart';
+part 'components/share_qr_code.dart';
```

Comment:
Why `part`? They don't use any private API, and are even located in a different folder.